manifests/0000_31_cluster-baremetal-operator_01_trusted-ca-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
@asalkeld should this configmap start off in the config directory? Some of the initial manifests have been added directly to the manifests/ like the Images configmap. But, since then we have always started with yaml in config/ that kubebuilder generates the manifest for and we later move it manifests/. I feel like we could go either ways with this configmap, so trying to get your opinion.